app/toolbar/toolbar.component.ts

```diff
@@ -96,6 +76,13 @@ export class ToolbarComponent implements OnInit {
         (data: any) => this.currentUser = <User> data,
         (error: any) => this.errorMessage = <any> error.error
       );
+      
+      this.notificationService.getUnreadNotificationsCount()
+        .then(
+          (response: any) => this.numberOfUnreadNotifications = response
+        ).catch(
+          (error: any) => console.log(error)
+        );
```

Comment:
Done by two conditions:
a) when after a notification is marked as read, the counter in the toolbar is decreased by the number of notifications marked as read (currently only 1 at once)
b) every minute the count is updated from the server-side such that the user even is informed about new notifications created on server-side